runtime/kusama/src/lib.rs

```diff
@@ -275,6 +275,7 @@ parameter_types! {
 	// 28 eras in which slashes can be cancelled (7 days).
 	pub const SlashDeferDuration: staking::EraIndex = 28;
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
+	pub const MaxNominatorRewardedPerValidator: u32 = 64;
```

Comment:
this number is a proposition as the one used in node-runtime in substrate, though I can't really tell about the precise value to use here.